### PR TITLE
feat: add homepage logic

### PR DESCRIPTION
**Pull request by dnlnrs**
- Added settings for default homepage (new tab landing)
- Change search engine url logic: if homepage is set, search url si available only in omnibox

### CODE REVIEW

**Comment by MystPi** [comment]
You removed the `searchurl` logic when creating tabs.

File: src/browser.js [hunk 1]
```diff
@@ -281,9 +288,9 @@ fetch('../package.json')
...
   .then(res => {
     version = res.version;
-    const item = localStorage.getItem('searchurl');
-    if (item) {
-      createTab('https://ninetails.cf/?v=' + version + '&e=' + item);
+    const homepageValue = localStorage.getItem('homepage');
+    if (homepageValue) {
+      createTab(homepageValue);
     } else {
       createTab('https://ninetails.cf/?v=' + version);
```
Comment (MystPi) [inline src/browser.js:295]: The `else` should contain something like https://github.com/MystPi/ninetails/blob/586e8fbd497a7407db31aaa314f1b6ec706a5051/src/browser.js#L285-L289

Comment (MystPi) [reply src/browser.js]: I fixed it.

Comment (dnlnrs) [reply src/browser.js]: Right! Thanks! ;)

**Pull request merged**